completion/booktabs.cwl

```diff
@@ -1,16 +1,20 @@
-# mode: booktabs.sty
+# package booktabs
 # dani/2006-02-18
-# muzimuzhi/2019-07-17
-\addlinespace
-\addlinespace[space]
+# muzimuzhi/17 Jul 2019
+# muzimuzhi/1 Sep 2019 for booktabs v1.618033
+
+\toprule
+\toprule[line width]
```

Comment:
an argument 'width' is recognized as latex-length, 'line width' is not.
So, please either revert to 'width' or 'line width%l'